### PR TITLE
Fixed a lot of typos

### PR DESCRIPTION
**Pull request by denzor200**

### CODE REVIEW

File: include/boost/pfr/io.hpp [hunk 1]
```diff
@@ -89,17 +89,17 @@ enable_istreamable_t<std::basic_istream<Char, Traits>, T> operator>>(std::basic_
 
 } // namespace detail
 
-/// IO manupulator to read/write \aggregate `value` using its IO stream operators or using \forcedlink{io_fields} if operators are not awailable.
+/// IO manipulator to read/write \aggregate `value` using its IO stream operators or using \forcedlink{io_fields} if operators are not awailable.
```
Comment (denzor200) [inline include/boost/pfr/io.hpp:92]: We can also fix `awailable` -> `available`

File: include/boost/pfr/io_fields.hpp [hunk 2]
```diff
@@ -21,7 +21,7 @@
 #include <boost/pfr/tuple_size.hpp>
 
 /// \file boost/pfr/io_fields.hpp
-/// Contains IO manupulator \forcedlink{io_fields} to read/write \aggregate `value` field-by-field.
+/// Contains IO manipulator \forcedlink{io_fields} to read/write an \aggregate field-by-field.
 ///
```
Comment (denzor200) [inline include/boost/pfr/io_fields.hpp:25]: fix `an` -> `a`

File: doc/pfr.qbk [hunk 3]
```diff
@@ -475,8 +475,8 @@ By default Boost.PFR [*auto-detects your compiler abilities] and automatically d
 The Boost.PFRs reflection has some limitations that depend on a C++ Standard and compiler capabilities:
 
 * Static variables are ignored
-* T must be aggregate initializable without empty base classes
```
Comment (apolukhin) [inline doc/pfr.qbk:478]: If I recall correctly, the old text was precise. Loophole implementation could work with aggregates that have non empty base classes

File: doc/pfr.qbk [hunk 4]
```diff
@@ -475,8 +475,8 @@ By default Boost.PFR [*auto-detects your compiler abilities] and automatically d
 The Boost.PFRs reflection has some limitations that depend on a C++ Standard and compiler capabilities:
```
Comment (apolukhin) [inline doc/pfr.qbk:475]: You've accidentally changed file permissions to 755 from 644

Please restore the permissions via ’git chmod -x the-affected-files’

**Review by apolukhin — CHANGES_REQUESTED** [review]

File: doc/pfr.qbk (hunk 3 above)
Comment (denzor200) [reply doc/pfr.qbk:478]: I'm sorry for the later answer! Here: https://godbolt.org/z/vWh5zjdqY
Should we fix it?

File: doc/pfr.qbk (hunk 4 above)
Comment (denzor200) [reply doc/pfr.qbk:475]: fixed

**Pull request merged**